src/picknik_ur_gazebo_config/README.md

```diff
@@ -1,5 +1,5 @@
 # picknik_ur_gazebo_config
 
-A MoveIt Studio site configuration for a UR5e simulated in Gazebo, based on the `picknik_ur_base_config` package.
+A MoveIt Studio site configuration for a UR5e simulated in Gazebo to show "scan and plan" applications, based on the `picknik_ur_gazebo_config` package.
```

Comment:
I think this change should be reverted, as this is the README for `picknik_ur_gazebo`.

Instead, this should be moved to the README for your new config package below.